### ISSUE TITLE
Different quotation mark for arcmin/arcsec in v3

### ISSUE THREAD

**Issue by taaem** [josephwright/siunitx#507]
In Siunitx V3 the arcsecond and arcminute quotation marks changed from using straight '/" to using the text formatted curly ones.

Since in the new handbook the quotation marks are correct, I suspect this is either an issue with my font (STIX2) or for some reason in my setup it chooses the wrong symbol.

If this is intended, would there be a way to force the old symbol to appear?

**Comment by josephwright** [comment]
@taaem Could you post a MWE? With the standard settings, unless you've explicitly redefined things, you should be getting the same as `\enuremath{{}'}`.

**Comment by taaem** [comment]
Here is a MWE, which needs to be compiled using lualatex.
```latex
\documentclass{scrartcl}

\usepackage{fontspec}
\usepackage[warnings-off={mathtools-colon,mathtools-overbracket},math-style=ISO, bold-style=ISO]{unicode-math}
\setmathfont{STIX Two Math}
\setmainfont{Stix Two Text}
\usepackage{siunitx}

\sisetup{detect-all}


\begin{document}
\qty{1.1}{\arcsecond}
\end{document}
```
The issue comes through using detect-all, which as I just saw was deprecated in v3, so I'll just remove it. But maybe a warning or an error should be shown if one tries to use it with v3?

**Comment by josephwright** [comment]
There is a warning for `detect-all`, but that's not the issue :)

There's a stray command in one part of the setup here that means in text mode the wrong path is being taken. I'll fix shortly.

**Comment by josephwright** [comment]
For the present, add
```latex
\DeclareSIUnit\arcminute{\ensuremath{{}'}}
\DeclareSIUnit\arcsecond{\ensuremath{{}''}}
```

**Comment by josephwright** [comment]
Fixed by 4c65f8e638c7e9834cd24948ef04ac636ab0eb92
